hash/spectrum_cass.xml

```diff
@@ -4867,6 +4879,72 @@ license:CC0
 	     Titles beginning with 'C'
 	     ############################################################################################################################## -->
 
+	<software name="cozumelsl">
+		<!--	Bug fixed release.	-->
```

Comment:
Hey @ICEknigh7, are you meaning to put tabs instead of spaces in some of these comments like this one? Not trying to be overly knitpicky, but srcclean will later convert these to spaces and it may look different from what you see. My PR right before yours is actually cleaning up a lot of multiple spaces -> single space in comments.

Reply:
Thanks for the heads up, didn't suspect that those would have their tabs automatically converted.

I've just converted all those one-line coments to multiple lines. Kind of makes things more readable and consistent, too.